apps/omg_status/lib/omg_status/metric/statix.ex

```diff
@@ -11,35 +11,180 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 defmodule OMG.Status.Metric.Statix do
   @moduledoc """
-  Useful for overwritting Statix behaviour.
+  Useful for overwritting Statix behaviour. The API doesn't change, but the underlying
+  connection can be swapped out to use `Kernel.send/2`
   """
   defmacro __using__(_opts) do
+    current_conn =
```

Comment:
Also not sure why we need all these additions from Statix's own implementation. Looks like the issue is `OMG.Status.Metric.Statix` is actually a stub https://github.com/omisego/elixir-omg/blob/master/apps/omg_status/lib/omg_status/metric/datadog.ex#L20-L25 so more ideal fix is probably rename the module to StatixStub or something similar.